cupy_setup_build.py

```diff
@@ -302,6 +301,20 @@
         })
 
 
+MODULES.append({
+    'name': 'random',
+    'file': [
+        ('cupy.random._bit_generators', ['cupy/random/cupy_distributions.cu']),
```

Comment:
How about adding this to `cuda` module? (around line 99)

Reply:
Yes, this is all WIP so I will be changing a lot of places where things are declared :)